guide/guide.html

```diff
@@ -780,7 +780,7 @@ <h3>Methods</h3>
 
 
 
-                    A method is an anonymous function which, when called, creates an object to store its locals and sets the local's proto pointer and its self slot to the target of the message. The Object method method() can be used to create methods. Example:
+                    A method is an anonymous function which, when called, creates an object to store its locals and sets the locals object's proto pointer and its self slot to the target of the message. The Object method method() can be used to create methods. Example:
```

Comment:
My understanding is that the ambient `locals` object's proto pointer is not changed, rather the `locals` object created by `method` "to store its locals" has its proto pointer set.